packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -77,14 +73,15 @@ export interface ParticipantItemProps {
   strings?: Partial<ParticipantItemStrings>;
 }
 
+const iconsContainerDefaultWidth = '5rem';
```

Comment:
can we not add this as a style in iconContainerStyle?

Reply:
sure :)